.travis.yml

```diff
@@ -60,10 +60,14 @@ install:
   - if [ "${TASK}" = 'compilepy3 ci-py3-unit' ] || [ "${TASK}" = 'ci-py3-integration' ]; then pip install "tox==3.0.0"; else make requirements; fi
   - if [ "${TASK}" = 'ci-unit' ] || [ "${TASK}" = 'ci-integration' ]; then pip install codecov; fi
   - if [ "${TASK}" = 'ci-unit' ] || [ "${TASK}" = 'ci-integration' ] || [ "${TASK}" = 'compilepy3 ci-py3-unit' ] || [ "${TASK}" = 'ci-py3-integration' ]; then sudo .circle/add-itest-user.sh; fi
+  # Note: We use ramdisk for better performance
+  - sudo -E ./scripts/travis/create-and-mount-ramdisk.sh
+  - ./scripts/travis/install-and-run-mongodb.sh
 
 # Let's enable rabbitmqadmin
 # See https://github.com/messagebus/lapine/wiki/Testing-on-Travis.
 before_script:
```

Comment:
For reference of editing existing mongo conf, this looks like a good hint: 
http://georgeshank.com/how-to-enable-a-mongodb-replica-set-on-travis/

```
- echo "foo = bar" | sudo tee -a /etc/mongodb.conf
- sudo service mongodb restart
```

which overall is more clean.